nss/src/passwd/mod.rs

```diff
@@ -140,23 +141,23 @@ fn passwd_entries_to_passwds(entries: Vec<PasswdEntry>) -> Vec<Passwd> {
     entries.into_iter().map(passwd_entry_to_passwd).collect()
 }
 
+static SSHD_BINARY_PATH: &str = "/usr/sbin/sshd";
```

Comment:
Can this be `const` instead?

Reply:
This is Rust, Marco... Everything is a `const` unless you explicitly say (by using `mut`) that it should be mutable :joy:

Reply:
Oh ok fair enough, I since the `const` keyword exists I thought it was clearer than `static`